src/udpstream.c

```diff
@@ -66,6 +67,31 @@ void *run_ntttcp_sender_udp4_stream(struct ntttcp_stream_client *sc)
 		(*(struct sockaddr_in *)&local_addr).sin_port = htons(client_port);
 		(*(struct sockaddr_in *)&local_addr).sin_family = sc->domain; /* AF_INET */
 
+		if(sc->source_addr_bind)
```

Comment:
let's simplify this new added code. some of the validation has been done when validating the parameters, so we don't need to validate here again